### PR TITLE
Pull Request for Issue2326: removing SSRL mono mask

### PR DESCRIPTION
**Pull request by helfrij**
Removed the BioXASSSRLMonochromatorMask class and Side- and Main- specific subclasses. Transferred all functionality to BioXASSSRLMonochromator.

### CODE REVIEW

**Comment by helfrij** [comment]
Hey @XuetaoLiu-CLS, would you review?

File: source/beamline/BioXAS/BioXASSSRLMonochromatorMask.cpp [hunk 1]
```diff
@@ -6,14 +6,10 @@ BioXASSSRLMonochromatorMask::BioXASSSRLMonochromatorMask(const QString &deviceNa
 {
 	// Initialize class variables.
 
-	upperBlade_ = 0;
```
Comment (XuetaoLiu-CLS) [inline source/beamline/BioXAS/BioXASSSRLMonochromatorMask.cpp]: do we still have this file? Isn't it deleted?

File: source/beamline/BioXAS/BioXASSSRLMonochromatorMask.h [hunk 2]
```diff
@@ -20,49 +20,6 @@ class BioXASSSRLMonochromatorMask : public BioXASBeamlineComponent
 
 	/// Returns true if the mask is connected, false otherwise.
 	virtual bool isConnected() const;
```
Comment (XuetaoLiu-CLS) [inline source/beamline/BioXAS/BioXASSSRLMonochromatorMask.h]: same question here

**Comment by XuetaoLiu-CLS** [comment]
Looks good other than a question on removed files.

Time Estimate: 5 minutes

File: source/beamline/BioXAS/BioXASSSRLMonochromatorMask.cpp (hunk 1 above)
Comment (helfrij) [inline source/beamline/BioXAS/BioXASSSRLMonochromatorMask.cpp]: Yup, you're right. Good catch!

**Comment by dretrex** [comment]
Looks good.
Time Estimate: 5 minutes

**Pull request merged**